cypress/component/integration/Password.cy.tsx

```diff
@@ -0,0 +1,116 @@
+import '../../../src/global.css';
+import { Password } from '../../../src/components';
+import { compareSnapshot } from '../../compareSnapshot';
+
+describe('Component | Integration | Password', function () {
+  describe('Input', () => {
+    describe('default password', () => {
+      beforeEach(() => {
+        cy.mount(
+          <div className="theme-dark">
```

Comment:
Are we testing theme style and colors to wrap in `theme-dark`? 🤔

Reply:
no we're not testing colors in these tests